__tests__/app.test.js

```diff
@@ -36,3 +36,43 @@ describe("GET: /api/categories", () => {
       });
   });
 });
+
+describe("GET: /api/reviews/:review_id", () => {
+  test("200: returns a review object, with correct properties", () => {
+    return request(app)
+      .get("/api/reviews/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual({
+          review: {
+            review_id: 1,
+            title: "Agricola",
+            category: "euro game",
+            designer: "Uwe Rosenberg",
+            owner: "mallionaire",
+            review_body: "Farmyard fun!",
+            review_img_url:
+              "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+            created_at: "2021-01-18T10:00:20.514Z",
+            votes: 1,
+          },
+        });
+      });
+  });
+  test("ERROR 404: returns bad path if wrong endpoint written", () => {
+    return request(app)
+      .get("/api/reviewz/1")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.message).toBe("Path not found");
+      });
+  });
+  test("ERROR 400: returns bad request if review_id does not exist", () => {
+    return request(app)
+      .get("/api/reviews/500")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.message).toBe("Bad Request - review_id does not exist");
```

Comment:
this shouldn't really be a `400` as it's a valid request to ask for review `500` - it just so happens it doesn't exist

Reply:
Edited to be a 404 - review not found.